### PR TITLE
Sh/updates

### PR DESCRIPTION
**Pull request by susuhahnml**

### CODE REVIEW

**Review by gschennersie — DISMISSED** [review]
For me everything looks ok and the test run without errors.

PS: Some minor point most probably unrelated to this branch: During checking your branch i also checked the Jupyter Notebooks and in usage\Configuration.ipynb and usage\Interactive.ipynb there is an error when i run the last cells at least on my computer. If you have any ideas to automatically test Notebooks let me know. We have this problem in other projects. And when visualizing configurations the classnames are in upper case (e.g. Frame) but for the kb they are in lowercase.

File: asp_interactive_configuration/DOC.md [hunk 1]
```diff
@@ -106,20 +106,20 @@ Constraints are defined using the `ooasp_cv/5` predicate:
 
 #### Complete constraints
```
Comment (rtaupe) [inline asp_interactive_configuration/DOC.md]: If I understand correctly, constraints previously called "complete" are now called "permanent", and constraints previously called "partial" are now called "potential". If that is true, all occurrences of the old terms should be updated (e.g., here).

Furthermore, I think the new terminology is an improvement (#7), but maybe we still find a better word than "potential", e.g., "temporary" (as an antonym to "permanent"). But for now we can keep "potential".

File: asp_interactive_configuration/ooasp/encodings_simple/ooasp.lp [hunk 2]
```diff
@@ -7,8 +7,8 @@
 ooasp_configuration(kb_name,config_name).
 #include "ooasp_aux_kb.lp". % Auxiliary predicates for the KB
 #external guess. % The guessing part is active
-#external check_partial_cv. % Checks for partial constraints apply
-#external check. % Checks for full constraints apply
+#external check_potential_cv. % Checks for potential constraints apply
+#external check_permanent_cv. % Checks for full constraints apply
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/encodings_simple/ooasp.lp]:
```suggestion
#external check_permanent_cv. % Checks for permanent constraints apply
```

File: asp_interactive_configuration/ooasp/tests/test_performance.py [hunk 3]
```diff
@@ -14,7 +14,13 @@ def test_solve_multiple_elements():
     for i in range(nr_of_elements):
         iconf.new_leaf("elementA")
     iconf.extend_domain(nr_of_elements + 5)
+    # print(iconf._statistics)
     config = iconf.next_solution()
+    # print(iconf._statistics)
+    iconf.check()
+    # print(iconf._statistics)
+
+
```
Comment (rtaupe) [inline asp_interactive_configuration/ooasp/tests/test_performance.py]: please remove temporarily needed code before merging

**Review by rtaupe — CHANGES_REQUESTED** [review]
Thank you, Susana!
Please let's just address a minor issue regarding new terminology before merging.

File: asp_interactive_configuration/ooasp/encodings_simple/ooasp.lp (hunk 2 above)
Comment (susuhahnml) [reply asp_interactive_configuration/ooasp/encodings_simple/ooasp.lp]: I added it in my commit

**Comment by susuhahnml** [comment]
All addressed in the last commit!

**Comment by rtaupe** [comment]
> All addressed in the last commit!

The place I pointed out where the old terminology is still used was just an example, there are also others. Do you want to fix them or shall I?

**Comment by susuhahnml** [comment]
I just updated the ones I could find, if you have any more in the radar, please add them :)

**Review by rtaupe — APPROVED** [review]

**Comment by rtaupe** [comment]
Thank you!

**Pull request merged**